site/en/blog/private-network-access-update-2023-02-02/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "Private Network Access update: Announce extension of the Deprecation Trial"
+authors:
+  - lyf
+description: Chrome is deprecating access to private network endpoints from non-secure public websites as part of the Private Network Access specification and is currently in a stage of deprecation trial until Chrome 113 at the moment.
+date: 2023-02-02
+hero: 'image/YLflGBAPWecgtKJLqCJHSzHqe2J2/dwtN0NkxkBmIz1EyhzAm.jpg'
+alt: 'Photo by Sara Cohen on Unsplash'
+tags:
+  - chrome-113
+  - security
+---
+
+Due to massive deprecation trial feedbacks mentioning the difficulty of deprecating HTTP
+private webpages and migrating those with TSL certification, we are now announcing an extension of the trial period until Chrome 113. Chrome 113 will roll out to Beta in April 2023 and Stable in May 2023.
+
```

Comment:
It seems to me that we should mention that we're building a prompt to help, even if we do not give all the details just yet. We can give an overview, point to relevant docs for the motivated reader, and say that we will put out a blog post with more details once it's ready?

Reply:
Done.